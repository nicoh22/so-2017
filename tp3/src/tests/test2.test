load tests/a1.txt tests/a2.txt tests/a3.txt tests/a4.txt tests/a5.txt
maximum
member auto 
member gato
member perro
member estoy
maximum
addAndInc bici
addAndInc bici
addAndInc bici
addAndInc bici
addAndInc bici
addAndInc bici
addAndInc bici
addAndInc bici
addAndInc bici
addAndInc bici
addAndInc bici
addAndInc bici
maximum
