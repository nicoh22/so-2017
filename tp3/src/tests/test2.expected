> El máximo es <auto,11>
> El string <auto> está
> El string <gato> está
> El string <perro> está
> El string <estoy> no está
> El máximo es <auto,11>
> El máximo es <bici,12>
